lighthouse-core/test/gather/gatherers/trace-elements-test.js

```diff
@@ -74,6 +74,14 @@ function makeLCPTraceEvent(nodeId) {
   };
 }
 
+it('startInstrumentation', async () => {
```

Comment:
since this is just checking that a simple function is called with a certain value, it seems to be a not useful test. If smokes test this protocol interaction sufficiently i suggest removing this unit test

Reply:
Ok